README.md

```diff
@@ -57,6 +57,84 @@ val sponsoredFundTransaction =
     wallet.fund(sourceAddress, destinationAddress, sponsorAddress = sourceAddress)
 ```
 
+### `Wallet.addAssetSupport()`
+
+```kotlin
+fun addAssetSupport(
+    sourceAddress: String,
+    assetCode: String,
+    assetIssuer: String,
+    trustLimit: String = "",
+    sponsorAddress: String = ""
+): Transaction
+```
```

Comment:
I would rename this to `addAsset()`, no need to use the term support.

Also, how are we thinking about liquidity pools? From a horizon perspective, you need a trustline to a liquidity pool just like you need a trustline to an asset. But the parameters are a bit different, so are we planning on adding a function for this like `addLiquidityPool()`?

Reply:
`addAssetSupport` is the name we have in the document now. We can change it, but we'll need to check with Bruno about it.

We haven't thought about liquidity pools yet, but most likely it will be a separate function. :+1: